app/controllers/filters/study_type_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Filters
+  class StudyTypeController < ApplicationController
+    def index; end
```

Comment:
These actions are more correctly `new` and `create` I think. It's a bit annoying because we need to match the C# URLs so we have to fudge the routes a bit.